cf_units/meta.yaml

```diff
@@ -11,7 +11,7 @@ build:
 
 requirements:
     build:
-        - python
+        - python <3
```

Comment:
Apparently this is problematic. It means that conda doesn't bake the python version :cry: - I think I need to fix that in conda-build...